.golangci.yml

```diff
@@ -102,6 +103,61 @@ linters:
           msg: Add a wrapper to nlwrap.Handle for EINTR handling and update the list in .golangci.yml.
       analyze-types: true
 
+    gocritic:
+      disabled-checks:
+        - appendAssign
+        - appendCombine
+        - assignOp
+        - badCall
+        - badLock
```

Comment:
Looks to be a single case; from the code, it looks like we're already suppressing linting; it's a bug dubious, but IIUC, we effectively only want to do a "try-lock" for synchronisation there 🤔  - could probably be an extra `nolint` (but may be good to add a comment);

```
daemon/logger/loggerutils/logfile.go:374:2: badLock: defer is missing, mutex is unlocked immediately (gocritic)
	w.rotateMu.Unlock() //nolint:staticcheck
	^
```